torch/fx/graph.py

```diff
@@ -387,6 +389,8 @@ def _gen_python_code(
 
         # Wrap string in list to pass by reference
         maybe_return_annotation : List[str] = ['']
+        include_stride = include_stride or (os.environ.get("ANNOTATE_STRIDE", "0") == "1")
```

Comment:
Perhaps call this something like FX_GRAPH_VERBOSE (it would be good to have FX_GRAPH in the env variable name)